mlir/tools/mlir-rocm-runner/mlir-rocm-runner.cpp

```diff
@@ -33,9 +32,23 @@
 #include "mlir/InitAllDialects.h"
 #include "mlir/Transforms/DialectConversion.h"
 
+#include "mlir/Conversion/GPUCommon/GPUCommonPass.h"
+#include "mlir/Dialect/GPU/Passes.h"
+#include "mlir/Pass/PassManager.h"
+#include "mlir/Support/FileUtilities.h"
+#include "mlir/Transforms/Passes.h"
+#include "mlir/Dialect/MIOpen/Generator/IsaNameParser.h"
+
 #include <cstdlib>
 #include <mutex>
 
+#if __INCLUDE_HIP__
```

Comment:
We can probably drop the `#if __INCLUDE_HIP_` in that this file won't be compiled if HIP isn't a dependency.